src/nacl/signing.py

```diff
@@ -17,7 +17,9 @@
 import six
 
 from nacl import encoding
+
 import nacl.bindings
+from nacl.public import PrivateKey, PublicKey
```

Comment:
That's a good point, I've submitted a [second PR](https://github.com/pyca/pynacl/pull/157) to make those changes.